core/data/pod/pod.proto

```diff
@@ -33,6 +33,7 @@ message Value {
     sint32 sint32 = 10;
     uint64 uint64 = 11;
     sint64 sint64 = 12;
+    uint64 handle = 29;
```

Comment:
Up until now it appears that this Value has mostly a 1:1 mapping between types (in go?) to fields. The `handle` is different, though, as it adds more semantics without a separate type.

What is the reason for not storing a handle in a uint64? Is this mainly to format them differently?

Have you considered adding a new (wrapper) type to make this type safe instead of using a plain uint64? This may also resolve some of the special cases that are added in the Go code to handle the new `handle` case, which can't rely on the type itself.

Also, it appears this is specifically a Vulkan handle. Would it make sense to be more explicit about the field/type and name it `vulkan_handle`? If we use `handle` in our codebase exclusively to refer to a Vulkan handle it may be fine as-is.

Either way, this probably warrants a comment to explain what this is and why it is somewhat different from the other fields here.